docs/lib/analytics/fragments/flutter/getting-started/30_initAnalytics.md

```diff
@@ -1,21 +1,15 @@
 Add the Auth and Analytics plugin, along with any other plugins you may have added as described in the *Prerequisites* section; 
 
 ```dart
-AmplifyAuthCognito authPlugin = AmplifyAuthCognito();
-AmplifyAnalyticsPinpoint analyticsPlugin = AmplifyAnalyticsPinpoint();
-
-Amplify amplifyInstance = Amplify();
-
-amplifyInstance.addPlugin(
-    authPlugins: [authPlugin], analyticsPlugins: [analyticsPlugin]);
+Amplify.addPlugins([AmplifyAuthCognito(), AmplifyAnalyticsPinpoint()]);
 ```
 
 Make sure that the amplifyconfiguration.dart file generated in the project setup is included and sent to Amplify.configure: 
 
 ```dart 
 import 'amplifyconfiguration.dart';
 
-Amplify.configure( amplifyconfig )
+Amplify.configure(amplifyConfig)
```

Comment:
This is the only code example where we are not showing the import statements for the packages. Should we add that?

Reply:
Good point, while checking it seems the getting started guide is all over the place. DataStore and API follow the analytics format and show the sample code, while others completely skip it (Storage, Auth)

I'll match it with DataStore/API for now, we may have to do some restructure later on.